foodsaving/notifications/models.py

```diff
@@ -17,7 +17,7 @@ class NotificationType(Enum):
     USER_BECAME_EDITOR = 'user_became_editor'
     YOU_BECAME_EDITOR = 'you_became_editor'
     FEEDBACK_POSSIBLE = 'feedback_possible'
-    NEW_STORE = 'new_store'
+    NEW_STORE = 'new_place'
```

Comment:
Could be left as `new_store` as type-specific notification

Reply:
I went with `NEW_STORE` --> `NEW_PLACE`